ml_metadata/tools/mlmd_bench/thread_runner.cc

```diff
@@ -86,15 +87,16 @@ tensorflow::Status ExecuteWorkload(const int64 work_items_start_index,
 }
 
 // Merges all the thread stats inside `thread_stats_list` into a workload stats
-// and reports the workload's performance according to that.
+// and reports the workload's performance according to that. Also, store the
+// performance inside `workload_summary`.
 void MergeThreadStatsAndReport(const std::string workload_name,
-                               ThreadStats thread_stats_list[], int64 size) {
+                               ThreadStats thread_stats_list[], int64 size,
+                               WorkloadConfigResult* workload_summary) {
```

Comment:
nit, given this is `void`, consider to simply return the workload_summary

Reply:
Yes, as for this one. I tried to return the workload_summary directly, however, since I cannot use `.CopyFrom()` to copy this workload summary directly (because this function-level workload_summary does not have workload_config initialized), I have to do `set_microseconds_per_operation()` and `set_bytes_per_second()` explicitly which I found too long. So, I keep the original design and pass in the `*benchmark.mlmd_bench_report().mutable_summaries(i)`. I am very sure about this one, wdyt? :)